style.css

```diff
@@ -0,0 +1,150 @@
+nav ul {
+    list-style-type: none;
+    display: grid;
+    grid-template: "sitename signup about" 30px / 60% 20% 20%;
+    background-color: lightcoral;
+    align-items: center;
+    font-family: 'Indie Flower', cursive;
+}
+
+section {
+  border-bottom: solid 3px;
+  font-family: 'Poor Story', cursive;
+}
+
+section:last-of-type, section:nth-of-type(5) {
+  border-bottom: none;
+}
+
+section:first-of-type {
+    text-align: center;
+    display: grid;
+    grid-template-rows: 100px 50px;
+    padding: 50px;
+}
+
+sub {
+  line-height: normal;
+  font-family: 'Indie Flower', cursive;
+  font-size: large;
+}
+
+section:first-of-type h1 {
+  align-self: end;
+  font-family: 'Ruslan Display', cursive;
+  font-size: xx-large;
+}
+
+section a {
+  text-decoration: none;
+  color: gray;
+  border-style: solid;
+  border-radius: 15px;
+  padding-top: 20px;
+  width: 200px;
+  height: 40px;
+  margin: auto;
+  vertical-align: middle;
+  background-color: thistle;
+}
+
+section a:hover {
+  background-color: maroon;
+}
+
+blockquote::before, blockquote::after {
+   content: "\"";
+}
+
+blockquote + p {
+  text-align: right;
+  padding-bottom: 20px;
+  font-family: 'Gamja Flower', cursive;
+  font-size: x-large;
+  padding-right: 50px;
+  margin: 0px;
+  background-color: lightseagreen;
+}
+
+blockquote + p::before {
+  content: "--";
+}
+
+blockquote {
+  padding: 50px;
+  font-family: 'Gamja Flower', cursive;
+  font-size: x-large;
+  background-color: lightseagreen;
+  margin: 0px;
+}
+
+h2 {
+  text-align: center;
+  padding: 10px;
+}
+
+section:nth-of-type(3) ul {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+}
+
+section:nth-of-type(3) ul li {
+  display: grid;
+  width: 300px;
+  height: 150px;
+  grid-template: "img x strong" 20px
+                  "img x text" 80px
+                  "blank x text" 50px
+                  / 100px 10px 200px;
+}
+
+section:nth-of-type(3) ul li p {
+  grid-area: text;
+}
+
+section:nth-of-type(3) ul li img {
+  grid-area img;
+}
+
+section:nth-of-type(3) ul li strong {
+  grid-area: strong;
+}
+
+section:nth-of-type(5) ul li {
+  display: grid;
+  text-align: center;
+  font-family: 'Indie Flower', cursive;
+  line-height: 2.0;
+  padding: 50px 100px 50px 100px;
+}
+
+section:nth-of-type(5) ul li img {
+  border-radius: 100px;
+}
+
+section:nth-of-type(5) ul, section:nth-of-type(6) ul {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+section:nth-of-type(6) ul li {
+  padding: 30px;
+}
+
+section:nth-of-type(7) {
+  display: grid;
+  grid-template-columns: 200px 200px;
+  padding-left: 50px;
+  background-color: lightcoral;
+}
+
+section:nth-of-type(7) ul {
```

Comment:
It looks like this was only one (or a few) commits. Making more commits would help you track changes and switch versions if need be.